bridge_adaptivity/module/engines/engine_vpal.py

```diff
@@ -1,6 +1,5 @@
 import logging
 import urlparse
-
```

Comment:
Please add empty line between standard library block and third-party libraries block